### ISSUE TITLE
Календарь должен начинаться с понедельника (при создании вакансии)

### ISSUE THREAD

**Issue by orestx** [msp4/client#81]
![image](https://cloud.githubusercontent.com/assets/2162824/13071672/ab65d45c-d49d-11e5-8e31-ac1c25f79362.png)

**Comment by vladmokryi** [comment]
Пофиксил на shopper

**Comment by alexzaporozhets** [comment]
@vladmokryi, насколько я помню, проблема возникла на client но фикс будет полездным для всех проектов

**Comment by vladmokryi** [comment]
да, я просто пометку сделал чтобы не забыть, таску не переводил

**Comment by vladmokryi** [comment]
Готово
